src/test/ui/codemap_tests/unicode_2.stderr

```diff
@@ -0,0 +1,18 @@
+error: invalid width `7` for integer literal
+  --> $DIR/unicode_2.rs:12:25
+   |
+12 |     let _ = ("a̐éö̲", 0u7);
+   |                     ^^^
+   |
+   = help: valid widths are 8, 16, 32, 64 and 128
+
+error: invalid width `42` for integer literal
+  --> $DIR/unicode_2.rs:13:20
+   |
+13 |     let _ = ("아あ", 1i42);
+   |                      ^^^^
```

Comment:
As per @est31's comment, could you add a case that points at the string itself to make sure regressions would get caught?